.github/workflows/e2e-test-kind.yaml

```diff
@@ -42,8 +42,8 @@ jobs:
       - name: Build Velero Image
         if: steps.image-cache.outputs.cache-hit != 'true'
         run: |
-          IMAGE=velero VERSION=pr-test make container
-          docker save velero:pr-test -o ./velero.tar
+          IMAGE=velero VERSION=pr-test BUILD_OUTPUT_TYPE=docker make container
```

Comment:
At present, CI E2E supports Linux only, opened a separate issue #8477 for requirements of running it in Windows.